tutorials/simulators/OpenFOAM.md

```diff
@@ -360,25 +262,109 @@ much more.
+        'rm constant/polyMesh/surfaceIndex',
+        'sed -i s/"ff_zMin"/"ff_SAVE"/g constant/polyMesh/boundary',
+        'sed -i s/"ff_zMax"/"ff_zMin"/g constant/polyMesh/boundary',
+        'sed -i s/"ff_SAVE"/"ff_zMax"/g constant/polyMesh/boundary',
+        'runApplication checkMesh',
+        'runApplication surfaceFeatureExtract',
+        'runApplication decomposePar -decomposeParDict system/decomposeParDict.SHM',
+        inductiva.commands.Command('snappyHexMesh -decomposeParDict system/decomposeParDict.SHM -dict system/snappyHexMeshDict -overwrite',mpi_config=config),
+        inductiva.commands.Command('checkMesh -decomposeParDict system/decomposeParDict.SHM -latestTime -meshQuality',mpi_config=config),
+        'runApplication reconstructParMesh -mergeTol 1e-08 -constant -latestTime',
+        'cp -r constant/polyMesh constant/polyMesh.origHalf',
+        'runApplication mirrorMesh -overwrite',
+        'runApplication topoSet -dict system/topoSetDict.faces.ff_zMin',
+        'runApplication createPatch -overwrite -dict system/createPatchDict.mirrorMesh',
+        'runApplication changeDictionary -constant -dict system/changeDictionaryDict.cyclicPatches -enableFunctionEntries',
+        'runApplication topoSet -dict system/topoSetDict.faces.cyclic',
+        'runApplication checkMesh -constant',
+        'rm -r processor*',
+        'rm -r constant/polyMesh.origHalf',
+        'cp -r 0.orig 0',
+        'cp system/controlDict.SHM system/controlDict',
+        'cp system/fvSchemes.SHM system/fvSchemes',
+        'cp system/fvSolution.SHM system/fvSolution',
+        'runApplication decomposePar',
+        inductiva.commands.Command('renumberMesh -overwrite',mpi_config=config),
+        'cp system/controlDict.SRS.init system/controlDict',
+        'cp system/fvSolution.SRS system/fvSolution',
+        'cp system/fvSchemes.SRS system/fvSchemes',
+        inductiva.commands.Command('applyBoundaryLayer -ybl 0.1',mpi_config=config),
+        'cp system/controlDict.SRS.init system/controlDict',
+        'cp system/fvSolution.SRS system/fvSolution',
+        'cp system/fvSchemes.SRS system/fvSchemes',
+        inductiva.commands.Command('rhoPimpleFoam',mpi_config=config),
+        'cp system/controlDict.SRS.avg system/controlDict',
+        'cp system/fvSolution.SRS system/fvSolution',
+        'cp system/fvSchemes.SRS system/fvSchemes',
+        inductiva.commands.Command('rhoPimpleFoam',mpi_config=config),
+        inductiva.commands.Command('postProcess -func sampleDict.surface.SRS -latestTime',mpi_config=config),
+    ]
+    ```
+3. **Start your Cluster**:
+
+    Now we need to start our cluster. This is done by:
+
+    ```python
+    mpi_cluster = inductiva.resources.MPICluster(
+                    machine_type="c3d-highcpu-180",
+                    data_disk_gb=300,
+                    num_machines=2)
+    mpi_cluster.start()
+    ```
+    
+4. **Run the simulation**:
+    All that is left is to run the simulation on the cluster.
```

Comment:
Finally, all that is left is to run the simulation on the configured MPI cluster: